modules/tinymce/src/themes/silver/main/ts/ReadOnly.ts

```diff
@@ -60,32 +49,37 @@ export const toggleToReadOnly = (uiComponents: RenderUiComponents, readonly: boo
   });
 };
 
-export const setupReadonlyModeSwitch = (editor: Editor, uiComponents: RenderUiComponents) => {
+const setupReadonlyModeSwitch = (editor: Editor, uiComponents: RenderUiComponents) => {
   editor.on('init', () => {
     // Force an update of the ui components disabled states if in readonly mode
-    if (editor.mode.isReadOnly()) {
+    if (editor.readonly) {
```

Comment:
Why was this changed back to `editor.readonly`? @jhaines specifically went through and made all these `editor.mode.isReadonly()` in a previous PR, as it's the definitive API for finding out if the editor is in readonly mode.

- [ ] Consider reverting this change